frontend/apps/reader/modules/readerfont.lua

```diff
@@ -209,11 +209,17 @@ end
 
 function ReaderFont:makeDefault(face)
     if face then
-        UIManager:show(ConfirmBox:new{
-            text = T( _("Set default font to %1?"), face),
-            ok_callback = function()
+        UIManager:show(MultiConfirmBox:new{
+            text = T( _([[Set %1 as default or fallback font? The fallback font ]] 
```

Comment:
if that's the problem, is it ok to put a long string here? but that would be ugly and in my understanding not good coding style.